readme.md

```diff
@@ -2,12 +2,10 @@
 
```

Comment:
- [ ] Great work but I don't see any change in the code files related to the API so I cannot proceed with a review of your code 🤔